packages/orbit-components/src/Stack/index.tsx

```diff
@@ -132,17 +139,19 @@ const Stack = (props: Props) => {
 
     return cx(
       typeof spaceAfter !== "undefined" && getSpaceAfterClasses(spaceAfter, viewport),
-      typeof align !== "undefined" && getAlignItemsClasses(align, viewport),
-      typeof align !== "undefined" && getAlignContentClasses(align, viewport),
-      typeof wrap !== "undefined" && getWrapClasses(wrap, viewport),
-      typeof grow !== "undefined" && getGrowClasses(grow, viewport),
-      typeof shrink !== "undefined" && getShrinkClasses(shrink, viewport),
-      typeof justify !== "undefined" && getJustifyClasses(justify, viewport),
-      getDirectionClasses(direction, viewport),
+      (flex || inline) && [
+        flex && "flex",
+        typeof align !== "undefined" && getAlignItemsClasses(align, viewport),
+        typeof align !== "undefined" && getAlignContentClasses(align, viewport),
+        typeof wrap !== "undefined" && getWrapClasses(wrap, viewport),
+        typeof grow !== "undefined" && getGrowClasses(grow, viewport),
+        typeof shrink !== "undefined" && getShrinkClasses(shrink, viewport),
+        typeof justify !== "undefined" && getJustifyClasses(justify, viewport),
+        getDirectionClasses(direction, viewport),
+      ],
       getSpacingClasses(spacing, viewport, direction, legacy),
-      inline && getDisplayInlineClass(inline, viewport),
+      inline != null && getDisplayInlineClass(inline, viewport),
       inline === false && "w-full",
```

Comment:
Does this have to explicitly be false? Can't it be undefined to apply `w-full`?

Reply:
Just missing this

Reply:
I think it's actually correct. Take a look on it [here](https://github.com/kiwicom/orbit/blob/dependabot/npm_and_yarn/remark-frontmatter-4.0.1/packages/orbit-components/src/Stack/helpers/getWidth.js#L5) 

So it has to be defined, but have a false 🤔

Reply:
Ok, thank you 👍